packages/create-block/lib/prompts.js

```diff
@@ -66,7 +61,7 @@ const dashicon = {
 	message:
 		'The dashicon to make it easier to identify your block (optional):',
 	validate( input ) {
-		if ( ! isEmpty( input ) && ! /^[a-z][a-z0-9\-]*$/.test( input ) ) {
```

Comment:
Why was `_.isEmpty` being used on a string? So odd.

Reply:
I don't think it was about the length then you could leave it up to the regular expression that requires a starting letter. I don't remember what `input` can resolve to, but I guess it might be better to check if the `input` is a string.

Reply:
FWIW, I've tried setting a `dashicon` to both `5` and an empty string and it always worked well, seems like it's always a string:

![Screenshot 2022-08-18 at 16 02 32](https://user-images.githubusercontent.com/8436925/185401458-8b679640-3426-464a-baa7-ea111636e92b.png)

Reply:
Great, thank you for confirming.